src/spyglass/common/common_dandi.py

```diff
@@ -223,12 +246,18 @@ def translate_name_to_dandi(folder):
     ----------
     folder : str
         location of files to be translated
+    danidset_dir : str
+        location of organized dandiset directory. If provided, will use this to
+        lookup the dandi_path for each file in the folder
 
     Returns
     -------
     dict
         dictionary of filename to dandi_path translations
     """
+    if dandiset_dir is not None:
```

Comment:
I don't love the structure of 'if option arg, return a diff func output'. That seems like a conditional that the parent should handle, if possible

I'm looking for where `translate_name` is invoked with the dir arg. I see it without on 184. Just future proofing?

Reply:
oops, I missed my commit for changing the function call.  I'll update the call to go to `lookup_dandi_translation in this case and leave `translate_name_to_dandi` in unused for now